### DISCUSSION TITLE
Resend member activation email/link?

### DISCUSSION THREAD

**Discussion by dambridge** [jacksleight/statamic-members#22] [Q&A]
When creating a new member, a link is sent via email to that user with a link to the Activate form. However, once that link expires, there doesn't appear to be a way to generate a new one, unless I'm missing something. Per the default copy in the activation email: 

`Activate your new account to access the members area. For your security, the link below expires after 1 hours. After that, please contact us for a new password.`

So is the expectation that, should a user's activation link expire before they can use it, they need to email the admin and the admin would what... send the password reset link instead?

**Comment by jacksleight** [comment]
> So is the expectation that, should a user's activation link expire before they can use it, they need to email the admin and the admin would what... send the password reset link instead?

Basically yes. The actual activation process is the same one used by Statamic when adding new users through the built-in user section, and just uses the rules you've set up in Laravel. You can extend the expiry date in `config.auth.passwords.*.expire`, or select different rules for activations and resets in `config.statamic.users.passwords`.

Once its expired either an admin has to send a password reset, or the user can request one from the login form. You could implement a "re-send activation link" option on your frontend. Memberbox doesn't currently provide this but it might be a nice addition?
